src/uct/ib/ud/accel/ud_mlx5.c

```diff
@@ -349,17 +319,24 @@ uct_ud_mlx5_ep_short_common(uct_ep_h tl_ep, uint8_t am_id,
                             uint32_t packet_flags, unsigned stat_ops_counter,
                             const char *func_name)
 {
+    uct_iov_t iov[2];
+
     UCT_CHECK_LENGTH(sizeof(uct_ud_neth_t) + header_size + data_size, 0,
                      uct_ud_mlx5_max_inline(), func_name);
+    iov[0].buffer = (void*)header;
+    iov[0].length = header_size;
+    iov[0].count  = 1;
```

Comment:
do we really need to set "count"? or it's unused? if unused - IMO better remove, for optimization

Reply:
We should set count because  it is required to calculate length of iov.

Reply:
ok